src/components/Documentation/Articles.js

```diff
@@ -72,20 +74,31 @@ const ArticleList = styled.ul`
   }
 `;
 
-const Article = ({ article = {}, depth }) => {
+const CustomList = styled.li`
+  ${ArticleLink} {
+    font-size: ${(props) => (props.depth > 0 ? "0.875rem" : "1rem")};
+    color: ${(props) =>
+      props.depth === 0 ? PALETTE.black80 : PALETTE.black60};
+    padding-left: ${(props) => (props.depth > 1 ? `${props.depth - 1}rem` : 0)};
+    font-weight: ${(props) =>
+      props.isActive ? FONT_WEIGHT.bold : FONT_WEIGHT.normal};
+  }
```

Comment:
we don't have to do this now, but at some pt we should figure out how to combine this with the styles for `NavItemEl` in /templates/ApiReference

Reply:
Good point. I can put it in `SharedStyle` folder tomorrow